plugins/org.yakindu.sct.doc.user/src/user-guide/simulating_statecharts.textile

```diff
@@ -210,64 +211,81 @@ h2(#advsim_debugging_with_breakpoints_and_snapshots). Debugging with breakpoints
 
 h3(#advsim_introduction). Introduction
 
-This advanced feature makes it possible to attach *breakpoints* to states and transitions. If a statechart simulation reaches a transition or a state with a breakpoint, it suspends execution of the simulation. A breakpoint can be amended with a _condition_ in order to suspend the simulation only if the condition is fulfilled (true).
+YAKINDU Statechart Tools' Professional Edition includes two simulator features that increase your productivity considerably when debugging statecharts:
+* You can attach *breakpoints* to states and transitions. If a statechart simulation reaches a transition or state with a breakpoint, it suspends execution of the simulation. A breakpoint can be amended with a _condition_. Such a breakpoint will suspend the simulation only if that condition is fulfilled, i.e., evaluates to _true_.
+* You can create *snapshots* of your statechart simulation. A snapshot contains everything making up your state machine simulation at any point in time. It describes the state of the state machine, if you will. Snapshots can be saved and restored later in order to continue the simulation from exactly the point the snapshot was taken.
```

Comment:
remove "if you will", this description is spot on and does not need to be put in question.